revolv/project/models.py

```diff
@@ -146,6 +145,12 @@ class Project(models.Model):
         max_length=255,
         help_text='How would you like to title this project?'
     )
+    tagline = models.CharField(
+        max_length=100,
+        null=True,
+        blank=False,
+        help_text='Select a short tag line that describes this project. (No more than 100 characters.)'
```

Comment:
I think you need blank=True in order for it to not be required.

https://docs.djangoproject.com/en/1.7/ref/models/fields/